### ISSUE TITLE
Does this lib support indexdb? 

### ISSUE THREAD

**Issue by pravin-d** [vincentracine/pocketjs#1]
I had a quick look at the code. Please correct me if I'm wrong, there seems to be no support for indexdb. Only localstorage and webSql is supported.

**Comment by vincentracine** [comment]
You are right. I am still working on offering IndexDB support but accidentally included it in the readme.md back when i started working on webSQL and IndexDB. Im sorry about this, I will update the readme and finish my work on offering IndexDB and then officially release it. Thanks for bringing this up!

On 4 Mar 2017, at 12:28, pravin-d <notifications@github.com<mailto:notifications@github.com>> wrote:


I had a quick look at the code. Please correct me if I'm wrong, there seems to be no support for indexdb. Only localstorage and webSql is supported.
